ci-operator/step-registry/cucushift/installer/check/vsphere/cpms/cucushift-installer-check-vsphere-cpms-commands.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+if test -f "${SHARED_DIR}/proxy-conf.sh"
+then
+    # shellcheck disable=SC1090
+    source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+
+function checkMultizone() {
+    check_result=0
+    cpms_network=$(oc get controlplanemachineset -n openshift-machine-api -ojson |  jq -r '.items[].spec.template.machines_v1beta1_machine_openshift_io.spec.providerSpec.value.network.devices')
```

Comment:
Can we also query the same key in `.items[].spec.template.machines_v1beta1_machine_openshift_io.spec.providerSpec.value.network.devices[].networkName` using `jq` just like in `checkSinglezone`

Reply:
for multizone. the network for different failure domain may different. so the value for devices level is null.
 
oc get controlplanemachineset -n openshift-machine-api -ojson |  jq -r '.items[].spec.template.machines_v1beta1_machine_openshift_io.spec.providerSpec.value.network'
{
  "devices": null
}